docs/antrea-ipam.md

```diff
@@ -106,6 +108,15 @@ Antrea deployment YAML are:
     ...
 ```
 
+Alternatively, when if Antrea is installed by helm, you can use the following
```

Comment:
```suggestion
Alternatively, when Antrea is installed by helm, you can use the following
```
or perhaps just remove "when Antrea is installed by helm" as the following words already says it's helm installation.

Reply:
ok, remove it